cmd/dep/ensure.go

```diff
@@ -224,7 +224,10 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 	// check if vendor exists, because if the locks are the same but
 	// vendor does not exist we should write vendor
 	var writeV bool
-	vendorExists, _ := dep.IsDir(filepath.Join(sw.Root, "vendor"))
+	path := filepath.Join(sw.Root, "vendor")
```

Comment:
Yeah, I could refactor this a bit to make it just one check instead of two. Working on that now.